src/commands/cli/release/build.ts

```diff
@@ -166,7 +167,7 @@ export default class build extends SfCommand<void> {
       // bump resolution deps
       if (flags.resolutions) {
         this.log('Bumping resolutions in the package.json to their "latest"');
-        repo.package.bumpResolutions('latest');
+        repo.package.packageJson.resolutions = repo.package.bumpResolutions('latest');
```

Comment:
this were all "void with side effects".  This one was simpler to return something and set that on the package